karma-shared.conf.js

```diff
@@ -189,6 +184,9 @@ module.exports = function(config, specificOptions) {
     config.sauceLabs.tunnelIdentifier = process.env.TRAVIS_JOB_NUMBER;
     config.sauceLabs.recordScreenshots = true;
 
+    // Try 'websocket' for a faster transmission first. Fallback to 'polling' if necessary.
+    config.transports = ['websocket', 'polling'];
+
```

Comment:
Is this not the default?

Reply:
According to the docs, the default is `['polling', 'websocket']` and it is not clear if ordering makes a difference, so I thought it wouldn't hurt to match angular/angular#27634.
But, yeah, this might be redundant.

Reply:
It seems that socket.io will 

> By default, a long-polling connection is established first, then upgraded to “better” transports (like WebSocket). 

So if I understand it, `['polling', 'websocket']` would connect with polling initially and then try to connect via a websocket. Seems strange but I guess that polling comes up quicker for short-lived connections?

Reply:
🤷‍♂️